.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Create new issue
+    url: https://new-issue.vuejs.org/
```

Thanks for the help! The issue link needs to be updated though. Each repository has a unique link.

```suggestion
    url: https://new-issue.vuejs.org/?repo=vuejs/vue-cli
```